packages/app/src/app/pages/Dashboard/Content/routes/PathedSandboxes/Folders.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import FolderEntry from '../../../Sidebar/SandboxesItem/FolderEntry';
+import getChildCollections from '../../../utils/getChildCollections';
+
+import { Folder, FoldersWrapper } from './elements';
+
+const getPath = name => {
```

Comment:
It's probably better to use `react-router`  for this (`props.match.params.path`). This may break if someone has a folder  called `sandboxes`.

Reply:
omg so much clean

```js
  const getPath = name => (params.path ? `${params.path}/${name}` : '/' + name);
```